src/components/ResultContainer.tsx

```diff
@@ -58,103 +67,218 @@ function ResultContainer({ response }: { response: QueryResponse | null }) {
     }
   }, [response]);
 
+  function convertURIToLabel(
+    type: string,
+    uri: string | string[] | null
+  ): string | string[] | null {
+    // Handle array of URIs
+    if (Array.isArray(uri)) {
+      return uri.map((singleUri) => convertURIToLabel(type, singleUri)).join(', ');
+    }
+
+    if (!uri) {
+      return uri;
+    }
+
+    switch (type) {
+      case 'sex': {
+        const entry = Object.entries(sexes).find(([, value]) => {
```

Comment:
```suggestion
        const [label, term_IRI] = Object.entries(sexes).find(([, term_IRI]) => {
```
1. This is complicated stuff, let's use clear and descriptive variable names!
2. Let's use destructuring on the `entry` here directly so I don't have to remember that the key is the label.

Reply:
Thanks to TS, I realized that we can't destruct on entry since `find` may return `undefined` and cause an error.